src/framework/preferences/messages/linksControls.json

```diff
@@ -1,3 +1,4 @@
 {
-    "linksControlsLabel": "Links & buttons"
+    "emphasisDescr": "Emphasize input options",
```

Comment:
@jobara: talked to @maozillah about this and she will make some changes based on our discussion